Code/GraphMol/Chirality.h

```diff
@@ -398,6 +398,15 @@ enantiomer" or "OR enantiomer". CIP labels, if present, are removed.
 RDKIT_GRAPHMOL_EXPORT void simplifyEnhancedStereo(
     ROMol &mol, bool removeAffectedStereoGroups = true);
 
+//! returns the meso centers in a molecule (if any)
+/*!
+ \param mol: molecule to work with
+
+*/
+RDKIT_GRAPHMOL_EXPORT std::vector<std::pair<unsigned int, unsigned int>>
+findMesoCenters(const ROMol &mol, bool includeIsotopes = true,
+                bool includeAtomMaps = false);
```

Comment:
This defaults to `false` here, and to `true` in several places in the Python wrappers files (both of them). We should probably default to the same thing on both places, and I think `false` makes more sense?

Reply:
good catch. I made the change in the Python wrapper for `findMesoCenters()`.
The other places where `includeAtomMaps=true` in the Python layer are for the canonical atom ranking code and there the python and C++ are consistent. Or did I miss something?

Reply:
No, it's fine. I was wondering whether we should be consistent with the ranking code too, and default to `true`, or default to `false`. I though `false` made more sense for `findMesoCenters()`, though.